backend/core/database.py

```diff
@@ -830,7 +836,7 @@ def _insert_bgp_updates(self):
             finally:
                 num_of_entries = len(self.insert_bgp_entries)
                 self.insert_bgp_entries.clear()
-                return num_of_entries
+            return num_of_entries
```

Comment:
why is this return pushed out of finally? What happens if the code throws an exception?